async/src/test/java/org/odk/collect/async/WorkerAdapterTest.kt

```diff
@@ -0,0 +1,74 @@
+package org.odk.collect.async
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.work.ListenableWorker
+import androidx.work.WorkerParameters
+import org.hamcrest.CoreMatchers.`is`
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.kotlin.any
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+import java.util.function.Supplier
+
+@RunWith(AndroidJUnit4::class)
```

Comment:
Does this test need Robolectric? I'm guessing something explodes without it?

Reply:
Yes I create an instance of `WorkerAdapter` in this test.